### PR TITLE
added explaination of [flow.json] in "Command-line usage" section

### PR DESCRIPTION
**Pull request by juggledad**
Added an explaination of how you can use the [flow.json] option to keep different undertakings in seperate files.

### CODE REVIEW

File: docs/writing-functions.md [hunk 1]
```diff
@@ -85,7 +85,18 @@ messages within the returned array. When multiple messages are returned for an
...
+
+{% highlight javascript %}
+var msg1 = { payload:"first out of output 1" };
+var msg2 = { payload:"second out of output 1" };
+var msg3 = { payload:"third out of output 1" };
+return [ [ msg1, msg2, msg3 ] ];
+{% endhighlight %}
+
+
+`Example 12:` In this example, `msg1`, `msg2`, `msg3` will be sent to the first output.
```
Comment (dceejay) [inline docs/writing-functions.md]: Example 12 ? and the words are the same as Example 1

File: docs/getting-started/running.md [hunk 2]
```diff
@@ -47,9 +47,12 @@ You can now create your [first flow](first-flow).
       -u, --userDir  DIR   use specified user directory
       -v                   enable verbose output
       -?, --help           show usage
+      [flows.json]         name of the data file you want to work with. Using different names, such as greenhouse.json
```
Comment (dceejay) [inline docs/getting-started/running.md]: As these lines are pre formatted the length matters.... 
image

**Comment by juggledad** [comment]
Hmmm, it looks like I was going to do something back in march (I don't rmember what) and when I did the current fork, it grabbed that one (?) and so pulled what I had done back then - at least that is my guess. Is there a way to destroy both forks so I can try again?

**Comment by dceejay** [comment]
You can either just delete the PR and start again - or just edit it back to where it should be - it will be flattened to just the deltas anyway.

**Pull request merged**